cmeutils/plot_tools.py

```diff
@@ -0,0 +1,21 @@
+import scipy.signal as signal
+
+
+def find_peaks(data):
+    """Finds peaks in 1-D data.
+    The peaks are any value greater than the average of all the data values. The data values are 1-D values (like x, y, or z) that     you wish to find the peaks of. 
+    Any points/values that excede the max height will be identified as peaks.
+    
+    Parameters
+    ----------
+    data : numpy.ndarray, shape (N,1)
+       Such as x, y, OR z.
+    Returns
+    ----------
+    tuple
+        The indices of peaks
+    """
+    total_peaks = signal.find_peaks(data)
+    avg_peaks = sum(data) / len(data)
+    peaks = signal.find_peaks(data, height=avg_peaks)
```

Comment:
I don't think we should assume that the peak height threshold is always going to be the average of data. The ideal height parameter will probably depend on what kind of data we're dealing with. I think this could be addressed by adding a kwargs parameter (as mentioned in my other comments).

Reply:
Good point! I will add a kwargs parameter for peak height. Thanks :)